test_runner/regress/test_pageserver_api.py

```diff
@@ -185,3 +190,146 @@ def test_pageserver_http_api_client_auth_enabled(neon_env_builder: NeonEnvBuilde
 
     with env.pageserver.http_client(auth_token=pageserver_token) as client:
         check_client(client, env.initial_tenant)
+
+
+def pageserver_predefined_scopes() -> List[PageserverLogScope]:
+    return [
+        TenantLogScope("debug", TenantId.generate()),
+        TimelineLogScope("debug", TenantId.generate(), TimelineId.generate()),
+    ]
+
+
+def pageserver_custom_scopes() -> List[PageserverLogScope]:
+    return [CustomLogScope("pageserver=debug", True), CustomLogScope("hyper=debug", True)]
+
+
+@pytest.mark.parametrize(
+    "pageserver_log_scope", pageserver_predefined_scopes() + pageserver_custom_scopes()
+)
+def test_pageserver_logs_toggle_overrides_env_var(
```

Comment:
worth having two timelines here and assert that we don't have the non debug level timeline logs while enabling logs for the second timeline

Reply:
Added.